src/class-wpml-gutenberg-strings-registration.php

```diff
@@ -96,6 +105,23 @@ private function update_string_location( array $package_data, stdClass $string_d
 		}
 	}
 
+	/**
+	 * Update string wrap tag.
+	 * Used for SEO, can contain (h1...h6, etc.)
+	 *
+	 * @param array    $package_data Package.
+	 * @param stdClass $string_data  String in the package.
+	 * @param string   $wrap_tag     String wrap.
+	 */
+	private function update_wrap_tag( $package_data, stdClass $string_data, $wrap_tag ) {
+		$string_id = apply_filters( 'wpml_string_id_from_package', 0, $package_data, $string_data->id, $string_data->value );
+		$string    = $this->string_factory->find_by_id( $string_id );
```

Comment:
@kagg-design Shouldn't you check the $string_id here instead of line 120?

Reply:
@brucepearson you are right, its overkill. find_by_id() always returns instance of WPML_ST_String::

Fixed it.